relayer-engine/src/storage/redis-storage.ts

```diff
@@ -154,7 +154,7 @@ export class RedisStorage implements Storage {
 
   async onTxSubmitted(vaaId: string, txId: string): Promise<void> {
     const txKey = `${vaaId}:txId`.toLowerCase()
-    await this.redis.set(txKey, txId)
+    await this.redis.set(txKey, txId, 'EX', 30 * 24 * 3600)
```

Comment:
Do we need 30 days cache here?

Reply:
The purpose of storing the tx id is to be able to return it to the bridge ui. The assumption here is that the user will execute a redemption within 30 days, but seems that 30 days might be a bit long. How about one week?

Reply:
Yes, I think one week is enough